app/assets/stylesheets/_index_admin.scss

```diff
@@ -0,0 +1,58 @@
+.admin-menu-container {
+  font-size: 20px;
+  color: black;
+  .menu-side {
+    display: flex;
+    .nav {
+      font-size: 30px;
+      justify-content: center;
+      border: 1px solid;
```

Comment:
Properties should be ordered alphabetically in order to have cleaner and easier way to search them.